test/integration/api/users.js

```diff
@@ -402,6 +402,12 @@ describe('api: /users', () => {
             asAlice.post('/v1/users/reset/verify')
               .send({ new: 'coolpassword' })
               .expect(403))));
+
+        it('should fail the request if email field is sent blank in request body', testService((service) =>
+          service.login('alice', (asAlice) =>
+            asAlice.post('/v1/users/reset/initiate')
+              .send({ email: '' })
+              .expect(400))));
```

Comment:
Would you mind also adding assertions about the response body? `code` should equal 400.2, and `details` should equal `{ field: 'email' }`.

Reply:
can do something like this ?
`it('should fail the request if email field is sent blank in request body', testService((service) =>
          service.login('alice', (asAlice) =>
            asAlice.post('/v1/users/reset/initiate')
              .send({ email: '' })
              .expect(400)
              .then(({ body: { code, details } }) => {
                details.should.eql({ field: 'email' });
                code.should.eql(400.2);
              }))));`